.github/workflows/ci.yml

```diff
@@ -24,10 +24,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up Ruby 2.7
+      - name: Set up Ruby 3.0
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7
+          ruby-version: 3.0
```

Comment:
Quote 3.0 to make it not become "3" in YAML stringification.

Reply:
I guess, we can use just the latest version for RuboCop. Anyway, it has its own `TargetRubyVersion`.

Reply:
Ah I always forget about that! Thank you @olleolleolle, and good point @AlexWayfer I'll use 3.3 here 👍

Reply:
> and good point @AlexWayfer I'll use 3.3 here 👍

I meant, you can use just `latest` here… I've checked [the syntax of `setup-ruby` action](https://github.com/ruby/setup-ruby#supported-version-syntax) — have not found `latest` word support, but you can use `ruby`, which means "MRI engine latest version".

But, what more effective, I believe, as I see in examples, you can just not specify `ruby-version:` option at all and it'll use the latest one. "Less code is a better code". :)

Reply:
mmmh let's give it a try. I'm a bit worried it might break unexpectedly at some point by pulling a new version, but as far as it pulls stable ones it should be fine? (at least until 4.0, but I guess we have time before that 😂)

Reply:
Got `input ruby-version needs to be specified if no .ruby-version or .tool-versions file exists`, trying with just `3` now which might give us what we want

Reply:
Yep, this will use the latest 3.x version available, just what we wanted 🙂
https://github.com/lostisland/faraday/actions/runs/7347680494/job/20004510675?pr=1544#step:3:17

Reply:
> mmmh let's give it a try. I'm a bit worried it might break unexpectedly at some point by pulling a new version, but as far as it pulls stable ones it should be fine? (at least until 4.0, but I guess we have time before that 😂)

I consider it as a safe warning that something has been changed and we should take some actions (it's a linter job, not test or build).



> Got `input ruby-version needs to be specified if no .ruby-version or .tool-versions file exists`

Got it.



> trying with just `3` now which might give us what we want

I'd still write `ruby`, which means `MRI`, and would be OK with that, even for possible 4.0 version. But, as you wish.

Reply:
> I consider it as a safe warning that something has been changed and we should take some actions

I'd be OK with that being just a warning, but in practice it will cause PRs to fail build, which is annoying for the contributor since it's unrelated to their changes.
This should work fine until we reach Ruby 4.0 so I'd say it's a good compromise 👍 
Thank you for the whole discussion, TIL quite a lot!